util/libcrypto.num

```diff
@@ -4817,8 +4817,14 @@ OSSL_CMP_CTX_get1_caPubs                4933	3_0_0	EXIST::FUNCTION:CMP
 OSSL_CMP_CTX_get1_extraCertsIn          4934	3_0_0	EXIST::FUNCTION:CMP
 OSSL_CMP_CTX_set1_transactionID         4935	3_0_0	EXIST::FUNCTION:CMP
 OSSL_CMP_CTX_set1_senderNonce           4936	3_0_0	EXIST::FUNCTION:CMP
-OSSL_CMP_log_open                       4937	3_0_0	EXIST::FUNCTION:CMP
-OSSL_CMP_log_close                      4938	3_0_0	EXIST::FUNCTION:CMP
-OSSL_CMP_print_errors_cb                4939	3_0_0	EXIST::FUNCTION:CMP
+OSSL_CMP_log_open                       4937	3_0_0	NOEXIST::FUNCTION:CMP
```

Comment:
looks like you may need to revert this file, rebase to master and make update again?

Reply:
I've done a 'make update'. Looks like it's fixed now.